modules/cnv-creating-vddk-image.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-importing-vmware-vm.adoc
+
+[id="cnv-creating-vddk-image_{context}"]
+= Creating the Virtual Disk Development Kit image
+
+You can download the VMware Virtual Disk Development Kit (VDDK) SDK, convert it into an image, and push it to a container repository.
+
+.Prerequisites
+
+* *My VMware* credentials to download the VDDK SDK
```

Comment:
s/*My VMware* credentials to download the VDDK SDK/You need your *My VMware* to download the VDDK SDK./

Reply:
Full sentences are unnecessary in a bullet list.